lisa/operating_system.py

```diff
@@ -179,7 +293,32 @@ def install_packages(
         if self._first_time_installation:
             self._first_time_installation = False
             self._initialize_package_installation()
-        self._install_packages(package_names)
+
+        self._install_packages(package_names, signed)
+
+    # package_exists
+    # Query if a package/tool is installed on the node.
+    # Return Value - bool
+    def package_exists(
+        self, package: Union[str, Tool, Type[Tool]], signed: bool = True
+    ) -> bool:
+        if isinstance(package, str):
```

Comment:
The code is same as install_packages, how about create a method called `_get_package_name` to share more code.

Reply:
Done